src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Ogooreck.Sample.BusinessLogic.FSharp.Tests.fsproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+
+        <IsPackable>false</IsPackable>
```

Comment:
not required in new stuff if you ref a test FW

Reply:
Sounds like the F# testing template in Rider should be updated 😎

Reply:
Yeah this was probably core 1/2 time